subjects/ai/nlp-scraper/audit/README.md

```diff
@@ -22,15 +22,15 @@ project
 
 ###### Does the structure of the project look like the above?
 
-###### Does the readme file give an introduction of the project, show the username, describe the feature engineering and show the best score on the leaderboard?
+###### Does the README file give an introduction of the project, show the username, describe the feature engineering and show the best score on the leaderboard?
```

Comment:
i see that is better to split this question to 4 questions.

Reply:
I will remove this question since it's actually not relevant to this project - as raised [here](https://github.com/01-edu/public/issues/2423)